### ISSUE TITLE
Improved issue-containing-element location finding

### ISSUE THREAD

**Issue by jschrab** [IBMa/equal-access#485]
### Summary
Currently, the Equal Access Accessibility Checker plugin will show a single DOM element that is responsible for the cited accessibility violation. In pages with large DOM structures, finding where this offending element exists can be time consuming at least and possibly ambiguous at worst. Being able to link from the plugin directly to the DOM element (within the browser's devtools) involved would be ideal.

### Justification
Having such a feature would very helpful in tracking down specific violations. Additionally, it would extend the Equal Access Accessibility Checker plugin to have increased feature parity with other accessibility testing browser plugins, such as [axe](https://www.deque.com/axe/browser-extensions/).

### "Must have" functionality
Currently, the "copy to clipboard" feature for Element Location is of lightweight usefulness. A link button to the devtools DOM tree would be more useful.

**Comment by tombrunet** [comment]
@jschrab See the "Accessibility Checker" sub-panel on the "Elements" main panel. Clicking an issue will jump to that element in the DOM. Is this what you're looking for?

**Comment by tombrunet** [comment]
Note that you can also select a node in the Element panel and filter the issues that are relevant to that DOM subtree.

**Comment by jschrab** [comment]
I ... had no idea that was there! I would say that does fit the need. Sorry to raise a non-issue! Thanks @tombrunet !

**Comment by tombrunet** [comment]
@jschrab No problem. We have a known discovery issue of that subpanel. Chrome won't let us open it automatically from the main panel, and it's kind of buried in the ... list of subtabs. We're still looking for better ways to help people find that.
